docs/openid-auth.md

```diff
@@ -0,0 +1,152 @@
+# Configuring OpenID Authentication
+
+The purpose of this guide is to demonstrate the process of configuring an OpenID Application as an authentication and authorization source for Code-Server.
```

Comment:
```suggestion
The purpose of this guide is to demonstrate the process of configuring an OpenID Application as an authentication and authorization source for code-server.
```

We refer to code-server in lowercase format, see: https://github.com/cdr/styleguide/blob/master/product-names.md#capitalized-terms